actions/hv.search.by.expression.yaml

```diff
@@ -0,0 +1,158 @@
+---
+description: Search for hypervisors with a selected expression
+enabled: true
+entry_point: src/openstack_actions.py
+name: hv.search.by.expression
+parameters:
+  timeout:
+    default: 5400
+  lib_entry_point:
+    default: workflows.search_by_expression.search_by_expression
+    immutable: true
+    type: string
+  query_type:
+    default: HypervisorQuery
+    immutable: true
+    type: string
+  as_admin:
+    default: True
+    immutable: true
+    type: boolean
+  cloud_account:
+    description: The clouds.yaml account to use whilst performing this action
+    required: true
+    type: string
+    default: "dev"
+    enum:
+      - "dev"
+      - "prod"
+  output_type:
+    default: "to_string"
+    type: string
+    enum:
+      - "to_html"
+      - "to_objects"
+      - "to_props"
+      - "to_string"
+    description: "
+    A string representing how to return the results of the query
+      - 'to_html' - a tabulate table (in html)
+      - 'to_props' - properties dicts as a python list
+      - 'to_objects - as a list of openstack resources
+      - 'to_string' - a tabulate table"
+    required: true
+  property_to_search_by:
+    default: "local_disk_free"
+    description: "choose property to search by (acts as OR for each)"
+    enum:
+      - "local_disk_free"
+      - "local_disk_size"
+      - "local_disk_used"
+      - "memory_free"
+      - "memory_size"
+      - "memory_used"
+      - "vcpus"
+      - "vcpus_used"
```

Comment:
Should 'runing_vms' and 'current_workload' be added here as well?

Reply:
I just went of the list in the task description: https://stfc.atlassian.net/browse/CLDSCRM-2002 - happy to add them in if needed!

Reply:
yea add them if they're integers